com.microsoft.mrtk.standardassets/Materials/CompressedButtonMaterial/MRTK_Stageplate_1cm.mat

```diff
@@ -0,0 +1,123 @@
+%YAML 1.1
```

Comment:
Stageplate probably shouldn't be in the button materials section.

Reply:
Adjusted file location